ddtrace/debugging/_signal/model.py

```diff
@@ -121,12 +122,17 @@ def _rate_limit_exceeded(self) -> bool:
             # We don't have a rate limiter, so no rate was exceeded.
             return False
 
-        exceeded = probe.limiter.limit() is RateLimitExceeded
+        exceeded = self.session is None and probe.limiter.limit() is RateLimitExceeded
         if exceeded:
             self.state = SignalState.SKIP_RATE
 
         return exceeded
 
+    @property
+    def session(self):
+        session_id = self.probe.tags.get("sessionId")
+        return Session.lookup(session_id) if session_id is not None else None
```

Comment:
if the session doesn't exists - the probe should not capture at all.

two points:
1. I think this code just make this manual probe as a normal probe if there is no sessions. 
2. if session level is 0 - it should also not capture.